micro-rdk/src/esp32/mod.rs

```diff
@@ -15,6 +15,7 @@ pub mod exec;
 pub mod hcsr04;
 pub mod i2c;
 pub mod pin;
+#[cfg(feature = "builtin-components")]
```

Comment:
are we sure about this one? It's really more of a peripheral than it is a component

Reply:
it gives clippy error with "--features esp32 --no-default-features"

Reply:
oh I see, nvm